src/views/Swap.vue

```diff
@@ -97,14 +97,17 @@
         {{ t('swap.insufficientAmount', { tokenSymbol: insufficientAmountTokenSymbol }) }}
       </template>
       <template v-else-if="isInsufficientBalance">
-        {{ t('exchange.insufficientBalance', { tokenSymbol: insufficientBalanceTokenSymbol }) }}
+        {{ t('exchange.insufficientBalance', { tokenSymbol: tokenFrom.symbol }) }}
```

Comment:
Could you check this place one more time please? We could have insufficient balance for fees too, so not sure that tokenFrom.symbol is enough here.

Reply:
The check below `isInsufficientXorForFee` handles this case

Reply:
Ok, cool))) thank you!